model/src/payout_curve.rs

```diff
@@ -86,6 +87,13 @@ impl Payouts {
             fee,
         )?;
 
+        if let InverseMaxPrice::OliviaMax = inverse_max_price_config {
+            let n_payouts = payouts.len() - 1;
+            let short_liquidation = payouts.get_mut(n_payouts).expect("several payouts");
+            short_liquidation.range =
+                *short_liquidation.range.start()..=maia_core::interval::MAX_PRICE_DEC;
```

Comment:
You did not have to adapt a test for this?

Reply:
Well, I didn't modify the inverse payout curve implementation. That's where most tests are located.

The `proptest` below still works after these changes because it doesn't check against hard-coded values.